README.md

```diff
@@ -22,33 +22,44 @@ Cette variable peut être définie dans le fichier [`.env`](.env) ou ses dériv
 \+ [Vue - Official](https://marketplace.visualstudio.com/items?itemName=Vue.volar)
 \+ [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
 
+### Installation de pnpm
+
+Ce projet utilise pnpm au lieu de npm. Si vous ne l'avez pas déjà installé :
+
+```sh
+# Enable pnpm via Corepack (inclus avec Node.js 20+)
+corepack enable pnpm
+```
+
+Il existe [d'autres méthodes d'installation si besoin](https://pnpm.io/installation).
+
 ### Initialisation du projet
 
 ```sh
-npm clean-install
+pnpm install
```

Comment:
I got the following warning:

> Warning 
> Ignored build scripts: @scarf/scarf, @sentry/cli, core-js, esbuild, vue-demi.            > Run "pnpm approve-builds" to pick which dependencies should be allowed to run scripts.

If I run the suggestion, I get:

> ? Choose which packages to build (Press <space> to select, <a> to toggle all, \<i> to invert selection) … 
❯ ○ @scarf/scarf
  ○ @sentry/cli
  ○ core-js
  ○ esbuild
  ○ vue-demi

Not sure what I'm supposed to do at this point. Might want to give instructions in the README.

Reply:
Nope. I thought it might be because I `brew install pnpm`, but just tried again with `corepack enable pnpm` and I get the same. Also corepack downloaded `pnpm-10.23.0.tgz` just now, not 10.24.0.